OrbitGl/Timer.h

```diff
@@ -5,32 +5,28 @@
 #ifndef ORBIT_GL_TIMER_H_
 #define ORBIT_GL_TIMER_H_
 
-#include <string>
-
-#include "GlUtils.h"
 #include "OrbitBase/Profiling.h"
 
 class Timer {
  public:
-  void Start() { start_ = MonotonicTimestampNs(); }
-
-  void Stop() { end_ = MonotonicTimestampNs(); }
+  explicit Timer() { Start(); }
 
+  void Start() { start_ = Now(); }
+  void Stop() { end_ = Now(); }
   void Reset() {
-    Stop();
-    Start();
+    start_ = Now();
+    end_ = 0;
   }
```

Comment:
I have to say that I find the interface slightly confusing, maybe because I compare it to how one would use a stopwatch in real life. Basically the constructor, `Start`, `Reset` all do kind of the same thing, but not really.  In particular `Start` and `Reset` are a bit weird as `Start` doesn't change `end_`, which could be older than `start_`, and `Reset` does what I would expect from `Start`.

Maybe what I would personally do is only have `Restart` (which would do what `Reset` does now) and `Stop`, and the constructor calls `Restart`.

Reply:
Yeah that's better.